seeds/clipper/arbitrum/clipper_arbitrum_trades_test_data.csv

```diff
@@ -0,0 +1,6 @@
+blockchain,project,version,block_date,block_time,token_bought_symbol,token_sold_symbol,token_pair,token_bought_amount,token_sold_amount,token_bought_amount_raw,token_sold_amount_raw,amount_usd,token_bought_address,token_sold_address,taker,maker,project_contract_address,tx_hash,tx_from,tx_to
+arbitrum,clipper,1,2022-12-15T00:00:00Z,2022-12-15T22:05:01Z,USDC,USDT,USDC-USDT,5.490002,5.49,5490002,5490000,5.491528220556,0xff970a61a04b1ca14834a43f5de4533ebddb5cc8,0xfd086bc7cd5c481dcc9c85ebe478a1c0b69fcbb9,0x52fb93047acf981c64a3d54f9fb63d1634293766,,0xe7b0ce0526fbe3969035a145c9e9691d4d9d216c,0x4e8b282691842b546869ff2ffb6e33e515d609bced31527bc049f92e6e00c831,0x52fb93047acf981c64a3d54f9fb63d1634293766,0xe7b0ce0526fbe3969035a145c9e9691d4d9d216c
+arbitrum,clipper,1,2022-12-15T00:00:00Z,2022-12-15T23:24:37Z,USDT,WETH,USDT-WETH,10.474675,0.0083,10474675,8300000000000000,10.485767680824999,0xfd086bc7cd5c481dcc9c85ebe478a1c0b69fcbb9,0x82af49447d8a07e3bd95bd0d56f35241523fbab1,0x90fa8de4f5f0058d3a8387903ed66d7b74072060,,0xe7b0ce0526fbe3969035a145c9e9691d4d9d216c,0x3c63866b0a50ef829cf1431b90116fad82e44823228dfa57a6fb43eda552304b,0x90fa8de4f5f0058d3a8387903ed66d7b74072060,0xe7b0ce0526fbe3969035a145c9e9691d4d9d216c
+arbitrum,clipper,1,2023-02-15T00:00:00Z,2023-02-15T11:00:35Z,DAI,USDT,DAI-USDT,114.45187511833501,114.56,114451875118335020000,114560000,114.46961515897834,0xda10009cbd5d07dd0cecc66161fc93d7c9000da1,0xfd086bc7cd5c481dcc9c85ebe478a1c0b69fcbb9,0xd358725d4fa42869acc1bc49e6326adb22828e29,,0xe7b0ce0526fbe3969035a145c9e9691d4d9d216c,0x52f3a5f2a8217e8bba343841bd379c076cfa06c1279d64ee43f891fc4aaedebc,0xd358725d4fa42869acc1bc49e6326adb22828e29,0xe7b0ce0526fbe3969035a145c9e9691d4d9d216c
+arbitrum,clipper,1,2022-12-17T00:00:00Z,2022-12-17T12:33:43Z,WETH,USDC,USDC-WETH,0.04963966456390679,58.644399,49639664563906790,58644399,58.71280245289767,0x82af49447d8a07e3bd95bd0d56f35241523fbab1,0xff970a61a04b1ca14834a43f5de4533ebddb5cc8,0xc160e4fe967f8ada7b58d9c578267ebbec1e7a63,,0xe7b0ce0526fbe3969035a145c9e9691d4d9d216c,0x283d63748d6390489926c305d148468c5308c75e58ea120fb0329a345b9c9447,0xb26a335d0357f316b70d5f7d894580bcdaac3343,0xdd94018f54e565dbfc939f7c44a16e163faab331
```

Comment:
Hi @amalashkevich, you can add the seeds in [dex_trades_seed.csv](https://github.com/duneanalytics/spellbook/blob/main/seeds/dex/trades/dex_trades_seed.csv). It would be easier for maintenance and writing tests.

Reply:
Done